apisix/core/etcd.lua

```diff
@@ -36,6 +36,7 @@ local function new()
     etcd_conf.host = nil
     etcd_conf.prefix = nil
     etcd_conf.protocol = "v3"
+    etcd_conf.api_prefix = "/v3"
```

Comment:
Where will it be used?

Reply:
In lua-resty-etcd. If api_prefix is not set, it would check the version of etcd to find the correct api_prefix to use in uri. Since we support etcd v3.4+ only, we could directly set it to avoid useless version check each time.